app/layout.tsx

```diff
@@ -32,7 +32,7 @@ export default function RootLayout({
           </main>
         </Providers>
       </body>
-      <GoogleAnalytics gaId={env.GOOGLE_ANALYTICS_TAG} />
+      <GoogleAnalytics gaId={env.NEXT_PUBLIC_GOOGLE_ANALYTICS_TAG} />
```

Comment:
only thing id suggest is making analytics ENV optional, and only including component if it exists, so app does not need it to specify env to run. not a huge blocker though